src/views/CollectionIndex/scss/index.scss

```diff
@@ -124,6 +124,8 @@
     margin: 30px 0 48px;
```

Comment:
For units like margin, padding, you should use the $spacer variable instead or use rem units to keep things consistent.